packages/perseus/src/strings.ts

```diff
@@ -505,7 +504,6 @@ export type PerseusStrings = {
         point2X: string;
         point2Y: string;
     }) => string;
-    // The above strings are used for interactive graph SR descriptions.
```

Comment:
I actually liked having this context, as the organization made it easier for our future selves to quickly find the chunk of strings! I'm not fussed either way, however as I know we have some great context on the strings themselves. :)